modules/web/src/app/wizard/step/cluster/template.html

```diff
@@ -490,100 +530,103 @@ <h4>IPv6</h4>
+                Enable Gateway API
+                <i class="km-icon-info km-pointer"
+                   matTooltip="Enable to use Gateway APIs. KKP will install the Gateway API CRDs in this cluster."></i>
+              </mat-checkbox>
+            </div>
+            }
+            }
+            @if (isKubernetesDashboardEnabled) {
+            <mat-checkbox [formControlName]="Controls.KubernetesDashboardEnabled">
               Kubernetes Dashboard
             </mat-checkbox>
-
+            }
             <mat-checkbox [formControlName]="Controls.OPAIntegration"
                           id="km-wizard-opa-integration-checkbox"
                           [matTooltipDisabled]="!form.get(Controls.OPAIntegration).disabled">
               <div fxLayout="row"
                    fxLayoutAlign=" center">
                 <span>OPA Integration</span>
-                <i *ngIf="isEnforced(Controls.OPAIntegration)"
-                   class="km-icon-info km-pointer"
+                @if (isEnforced(Controls.OPAIntegration)) {
+                <i class="km-icon-info km-pointer"
                    matTooltip="OPA Integration is {{form.get(Controls.OPAIntegration).value ? 'enforced' : 'disabled'}} by your admin."></i>
+                }
                 <i class="km-icon-warning km-pointer"
                    matTooltip="OPA (Open Policy Agent) has been deprecated in KKP 2.28 and will be removed in the future, Kyverno has replaced it as an Enterprise Edition feature for policy management.."></i>
               </div>
             </mat-checkbox>
-
-            <mat-checkbox *ngIf="isEnterpriseEdition"
-                          [formControlName]="Controls.KyvernoIntegration">
+            @if (isEnterpriseEdition) {
+            <mat-checkbox [formControlName]="Controls.KyvernoIntegration">
               Kyverno Policy Management
               <i class="km-icon-info km-pointer"
                  matTooltip="Enable to deploy Kyverno for this User Cluster"></i>
             </mat-checkbox>
-
-            <ng-container *ngIf="isMLAEnabled">
-              <mat-checkbox [formControlName]="Controls.MLALogging">
-                User Cluster Logging
-                <i *ngIf="isEnforced(Controls.MLALogging)"
-                   class="km-icon-info km-pointer"
-                   matTooltip="User Cluster Logging is enforced by your admin."></i>
-              </mat-checkbox>
-
-              <mat-checkbox [formControlName]="Controls.MLAMonitoring">
-                User Cluster Monitoring
-                <i *ngIf="isEnforced(Controls.MLAMonitoring)"
-                   class="km-icon-info km-pointer"
-                   matTooltip="User Cluster Monitoring is enforced by your admin."></i>
-              </mat-checkbox>
-            </ng-container>
-
-            <mat-checkbox *ngIf="isUserSshKeyEnabled"
-                          [formControlName]="Controls.UserSSHKeyAgent">
+            }
+            @if (isMLAEnabled) {
```

Comment:
https://github.com/kubermatic/dashboard/pull/7681/files

I need to re-run migration with migration schematic to so that we can cater fix which was merged later